helm-charts/orchestrator/values.yaml

```diff
@@ -56,27 +56,27 @@ rhdhPlugins: # RHDH plugins required for the Orchestrator
   npmRegistry: "https://npm.stage.registry.redhat.com" # NPM registry is defined already in the container, but sometimes the registry need to be modified to use different versions of the plugin, for example: staging(https://npm.stage.registry.redhat.com) or development repositories
```

Comment:
Once the plugins are verified will we need another build with this set to `https://npm.registry.redhat.com`